meteor/lib/lib.ts

```diff
@@ -1412,3 +1412,12 @@ export function equivalentArrays<T>(a: T[], b: T[]): boolean {
 	}
 	return true
 }
+
+export function equalArrays<T>(a: T[], b: T[]): boolean {
```

Comment:
Minor note: Since this function doesn't do deep comparisons, `T` should perhaps be limited to primitives `<T extends string | number | undefined | null | boolean>`?

It might also be good to document this function to motivate its existence (compared to the more common `_.isEqual`)

Reply:
The motivation is that for non-deep comparisons, this implementation is 2x faster than _.isEqual on a worst-case scenario:

![obraz](https://user-images.githubusercontent.com/3635991/96236914-eec2f100-0f9c-11eb-93a8-267b944cdeaa.png)

and an order of magnitude faster in a best-case scenario (length of array is different):

![obraz](https://user-images.githubusercontent.com/3635991/96237743-f5059d00-0f9d-11eb-9c60-cd94cefe6f87.png)

I think I would like to keep it as-is, and just be a shallow equal, but allow using it on arrays of objects - that's very useful for the UI when it needs to compare incoming props that may be produced by some sort of a `.map` or `.filter` function, for optimizations - in that case, the incoming array is a new array, but the objects inside are the same.